Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -122,20 +142,27 @@ private extension LayoutManager {
 
         return blockquoteRect
     }
+    
+    private func drawBlockquoteBorder(in rect: CGRect, with context: CGContext, at depth: Int) {
+        
+        let quoteCount = blockquoteBorderColors.count
+        let index = min(depth, quoteCount-1)
+        
+        if blockquoteBorderColors.indices.contains(index) {
```

Comment:
this could be rewritten like this:
```
guard index < blockquoteBorderColors.count else {
  return
}
```

Reply:
thats true, I guess is that more performant than `indices.contains` also? (although its a small array)

I was trying to keep similar style to the `func drawBlockquoteBackground` on line 158 which uses an `if let`.

but I guess it doesn't really matter and actually I think a guard makes more sense for both.